src/open_inwoner/components/templatetags/icon_tags.py

```diff
@@ -12,7 +12,7 @@ def icon(icon, **kwargs):
     Fontawesome Brands: https://fontawesome.com/v5.15/icons?d=gallery&p=2&s=brands
 
     Usage:
-        {% icon "arrow-forward" %}
+        {% icon "arrow-forward" outlined=True %}
```

Comment:
I know this is just documentation, but this way it reminds other developers to not forget we use the outlined (not filled) versions.